### PR TITLE
Important lore characters

### PR DESCRIPTION
**Pull request by Loralius**
## Changelog:
- Added plot armour to cultural leaders and notables.

## Developer changelog:
Affected characters: 
`Alleria Anub'arak Baine Brann Broxigar Cairne Cenarius Chen Erazmin Falric Gallywix Gazlowe Gelbin Hogger Jarod Kargath Kazra'jin Kurdran Lili Maiev Malakk Marwyn Moira Muradin Nazgrel Rastakhan Remulos Rivendare Rokhan Shek'zeer Sul Talanji Taran Teron'gor Tiboan Ukorz Valeera Varok Vashj Vereesa Zalazane Zhu Zul`

`Edwin, Vanessa; Odyn, Thorim, Loken, Ra; Sicco; Hamuul, Magatha; Taretha, Blackmoore;`
`Rivendare; Malkorok, Zuluhed, Nekros; Jakra'zet, Yazma; Alonsus, Bolvar, Thassarian, Saidan, Alexandros, Darion, Mathias; Arthur, Meredith, Lucille Waycrest; Lord Stormsong; Mug'thol; Sintharia, Arygos; Alzzin; Phaoris, Tekahn, Ammantep, Amet; Lunara; Kiro; Korthek, Vorrik; Ulfar; Salyis;`

## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [x] The mod takes less than 5.5 GB in the Task Manager (Windows)

### CODE REVIEW

File: history/characters/76000_nerubian.txt [hunk 1]
```diff
@@ -10,7 +10,8 @@
 	# disallow_random_traits = yes
 	# trait=intellect_good_3 trait=wrathful trait=diligent trait=sadistic trait=brave
 	# add_strong_claim = k_amonkahet add_strong_claim = k_duatkahet
-	# 395.4.5={ birth=yes add_spouse = 76001 }
+	# 395.4.5={ birth=yes add_spouse = 76001 
+		effect = { make_important_lore_character_effect = yes } }
```
Comment (zumbak04) [inline history/characters/76000_nerubian.txt:14]: It's not commented thus it causes errors.

File: history/characters/16000_gnome.txt [hunk 2]
```diff
@@ -9,6 +9,7 @@
 	trait=education_learning_4
```
Comment (zumbak04) [inline history/characters/16000_gnome.txt:9]: Check `Crusader Kings III\logs\error.log` errors. Look for errors mentioning `history\characters`.

Comment (Loralius) [reply history/characters/16000_gnome.txt:9]: Resolved all errors returned from search of "make_important_lore_characters_effect" and "important". Other history/characters errors unrelated.

File: history/characters/10000_orc.txt [hunk 3]
```diff
@@ -419,7 +420,8 @@
 	martial=6 diplomacy=5 stewardship=4 intrigue=6 learning=6
 	trait=education_learning_3 
 	trait=sadistic trait=wrathful
-	536.1.1={ birth=yes trait=creature_orc }
+	536.1.1={ birth=yes trait=creature_orc
+			effect = { make_important_lore_character_effect = yes } }
```
Comment (zumbak04) [inline history/characters/10000_orc.txt:424]: Tab rules. The closing bracket must be on the same tab level as the opening bracket.
```suggestion
	536.1.1={
		birth=yes trait=creature_orc
		effect = { make_important_lore_character_effect = yes }
	}
```

**Review by zumbak04 — CHANGES_REQUESTED** [review]

**Comment by arithon** [comment]
Plot armour as in can't be assassinated etc?

File: history/characters/10000_orc.txt (hunk 3 above)
Comment (Loralius) [reply history/characters/10000_orc.txt]: Should be good now. I also standardised the formatting for some pre-existing effect markers.

**Comment by Loralius** [comment]
> Plot armour as in can't be assassinated etc?

This PR is pursuant to #192 but I did not link it for closure so as to keep the issue open-ended for future additions if necessary.
My code does not define the effects of plot armour. Rather, it inserts the markers into character history files. 
This is what plot armour does: (effect lasts for 5 years beginning from game start date)
image

File: history/characters/10000_orc.txt [hunk 4]
```diff
@@ -10,7 +10,9 @@
 	trait=wrathful trait=greedy trait=arrogant trait=arbitrary
 	disallow_random_traits = yes
 	father = 10701 #Ugratil
-	542.1.13={ birth=yes trait=creature_orc effect = { make_important_lore_character_effect = yes } }
+	542.1.13={ birth=yes trait=creature_orc 
+		effect = { make_important_lore_character_effect = yes }
+	}
```
Comment (zumbak04) [inline history/characters/10000_orc.txt:15]: Tab rules. If the opening and closing brackets aren't on the same line, the code between them goes on the next lines with a tabulation.
```suggestion
	542.1.13={
		birth=yes trait=creature_orc 
		effect = { make_important_lore_character_effect = yes }
	}
```
You may read it here, it's called K&R style.
https://en.wikipedia.org/wiki/Indentation_style#Brace_placement_in_compound_statements

File: history/characters/10000_orc.txt [hunk 5]
```diff
@@ -291,7 +309,10 @@
 	# trait=education_martial_4 trait=sadistic
 	# trait=diligent trait=wrathful trait=ambitious 
 	# disallow_random_traits = yes
-	# 553.6.18={ birth=yes trait=creature_orc }
+	# 553.6.18={ 
+		birth=yes trait=creature_orc
+		# effect = { make_important_lore_character_effect = yes }
+	#}
```
Comment (zumbak04) [inline history/characters/10000_orc.txt:315]: Check `error.log` before requesting reviews. One small typo can break everything.
```suggestion
	# 553.6.18={ 
		# birth=yes trait=creature_orc
		# effect = { make_important_lore_character_effect = yes }
	#}
```
That's `error.log`
```
Line 699: [00:16:14][jomini_effect.cpp:557]: Unknown effect martial at  file: history/characters/10000_orc.txt line: 342
	Line 700: [00:16:14][jomini_effect.cpp:557]: Unknown effect diplomacy at  file: history/characters/10000_orc.txt line: 342
	Line 701: [00:16:14][jomini_effect.cpp:557]: Unknown effect stewardship at  file: history/characters/10000_orc.txt line: 342
	Line 702: [00:16:14][jomini_effect.cpp:557]: Unknown effect intrigue at  file: history/characters/10000_orc.txt line: 342
	Line 704: [00:16:14][jomini_effect.cpp:557]: Unknown effect learning at  file: history/characters/10000_orc.txt line: 342
	Line 707: [00:16:14][jomini_effect.cpp:557]: Unknown effect 511.8.11 at  file: history/characters/10000_orc.txt line: 345
	Line 709: [00:16:14][jomini_effect.cpp:557]: Unknown effect 511.8.11 at  file: history/characters/10000_orc.txt line: 349
	Line 711: [00:16:14][jomini_effect.cpp:557]: Unknown effect 531.8.11 at  file: history/characters/10000_orc.txt line: 357
	Line 713: [00:16:14][jomini_effect.cpp:557]: Unknown effect 591.11.1 at  file: history/characters/10000_orc.txt line: 358
	Line 715: [00:16:14][jomini_effect.cpp:557]: Unknown effect 612.12.2 at  file: history/characters/10000_orc.txt line: 369
	Line 717: [00:16:14][jomini_effect.cpp:557]: Unknown effect dna at  file: history/characters/10000_orc.txt line: 375
	Line 718: [00:16:14][jomini_effect.cpp:557]: Unknown effect martial at  file: history/characters/10000_orc.txt line: 378
	Line 720: [00:16:14][jomini_effect.cpp:557]: Unknown effect diplomacy at  file: history/characters/10000_orc.txt line: 378
	Line 721: [00:16:14][jomini_effect.cpp:557]: Unknown effect stewardship at  file: history/characters/10000_orc.txt line: 378
```

**Review by zumbak04 — CHANGES_REQUESTED** [review]

**Review by zumbak04 — APPROVED** [review]
LGTM! Great job! However, I think Thassarian isn't an important character.  However, I don't mind your vision.

**Review by TheDrPibb1999 — APPROVED** [review]
All good for me

**Pull request merged**